src/components/HearingCard.js

```diff
@@ -7,22 +7,25 @@ import LabelList from './LabelList';
 import getAttr from '../utils/getAttr';
 import {getHearingURL, getHearingMainImageURL} from '../utils/hearing';
 import moment from 'moment';
+import config from '../config';
 
 const HearingCard = ({hearing, language, className = ''}) => {
   const backgroundImage = getHearingMainImageURL(hearing);
   const imageStyle = {
     backgroundImage: backgroundImage ? `url(${backgroundImage})` : ''
   };
   // FIXME: Should there be direct linking to hearing using certain language?
-  const translationAvailable = !!getAttr(hearing.title, language/* , {exact: true} */);
+  const translationAvailable = !!getAttr(hearing.title, language, {exact: true});
   const expiresSoon = moment(hearing.close_at).diff(moment(), 'weeks') < 1;
+  const availableInLanguageMessages = { fi: 'Kuuleminen saatavilla Suomeksi', sv: 'Frågorna tillgängliga', en: 'Questions available in English'};
```

Comment:
Fix the text to `{ fi: 'Kuuleminen saatavilla suomeksi', sv: 'Enkäten tillgänglig på svenska', en: 'Questionnaire available in English'}`